index.js

```diff
@@ -46,7 +47,18 @@ async function run() {
     await exec.exec(`${pkgManager} run build ${buildArgs}`, []);
     console.log("Finished building your site.");
 
-    await exec.exec(`${pkgManager} run scully -- --nw ${scullyArgs}`, []);
+    // determine the scully version
+    const packageJsonRaw = await fs.readFile('./package.json', 'utf8');
```

Comment:
Maybe we could parse or look for the package-lock/yarn.lock file?

Then the version value there would not need to be scrubbed of `~` or `^`.

Reply:
ah yes, that would probably a good idea. Youst thinking about if it makes sense when one does not check in the lock files? I think as of `npm install` in the CLI, an old version could be used?

Reply:
ah no. `npm i` / `yarn i` should update the lock file before.

Reply:
Oh good call. You had me wondering too lol.